dsc/docs-conceptual/dsc-2.0/concepts/class-based-resources.md

```diff
@@ -835,25 +957,27 @@ method or function and move the code there instead. This makes testing and maint
 Resource easier. It also makes your DSC Resource's methods easier to read and understand.
 
 <!-- Reference Links -->
-
-[01]: /powershell/module/microsoft.powershell.core/about/about_classes
-[02]: /powershell/module/microsoft.powershell.core/about/about_module_manifests#rootmodule
-[03]: /powershell/module/microsoft.powershell.core/about/about_module_manifests#nestedmodules
-[04]: /powershell/module/microsoft.powershell.core/about/about_module_manifests#dscresourcestoexport
-[05]: /powershell/module/psdesiredstateconfiguration/get-dscresource
-[06]: /powershell/module/microsoft.powershell.core/new-modulemanifest
-[07]: /powershell/module/microsoft.powershell.core/about/about_module_manifests
-[08]: /dotnet/api/system.management.automation.dscresourceattribute
-[09]: /dotnet/api/system.management.automation.dscresourceattribute.runascredential
-[10]: /powershell/dsc/configurations/runasuser?view=dsc-1.1&preserve-view=true
-[11]: /dotnet/api/system.management.automation.dscpropertyattribute
-[12]: /powershell/module/microsoft.powershell.core/about/about_functions_advanced_parameters#parameter-and-variable-validation-attributes
-[13]: /powershell/module/microsoft.powershell.core/about/about_preference_variables#erroractionpreference
-[14]: /powershell/module/microsoft.powershell.core/about/about_enum
-[15]: /powershell/module/microsoft.powershell.core/about/about_classes#class-properties
-[16]: /powershell/module/microsoft.powershell.core/about/about_classes#class-methods
-[17]: /powershell/module/microsoft.powershell.core/about/about_classes#constructor
-[18]: ../how-tos/resources/authoring/checklist.md
-[19]: #complex-properties
-[20]: #enum-properties
-[21]: /dotnet/api/system.management.automation.validateargumentsattribute
+[01]: ../how-tos/resources/authoring/checklist.md
+[02]: /dotnet/api/system.management.automation.dscpropertyattribute
```

Comment:
Looks like you used my new script. Nice!

Reply:
It worked really well!